.github/workflows/coverage.yml

```diff
@@ -66,7 +66,7 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: coverage
-          path: coverage/
+          path: coverage/merged
```

Comment:
We should not change the default here because our other repos also use this workflow (i.e. this is a breaking change!)